README.md

```diff
@@ -26,12 +26,23 @@ Install-Package SimplerSoftware.EntityFrameworkCore.SqlServer.NodaTime
 
 Then call the `UseNodaTime()` method as part of your SqlServer configuration during the `UseSqlServer` method call:
 ```csharp
-using Microsoft.EntityFrameworkCore.SqlServer.NodaTime.Extensions;
+using Microsoft.EntityFrameworkCore.SqlServer;
```

Comment:
should this not simply be: using Microsoft.EntityFrameworkCore ?

Reply:
Doh!  Good catch. Thanks!